bindings/cpp/CMakeLists.txt

```diff
@@ -171,7 +170,7 @@ endif()
 
 # Platform-specific test configuration
 if(WIN32)
-    target_link_libraries(opendal_cpp userenv ws2_32 bcrypt)
+    target_link_libraries(opendal_cpp PRIVATE userenv ws2_32 bcrypt Ntdll.lib)
```

Comment:
System libraries (such as userenv/ws2_32) should use PUBLIC, as they are interface dependencies, while internal libraries (Ntdll.lib) should use PRIVATE.